lib/src/widget/scale_text.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+
+class ScaleText extends StatelessWidget {
+  const ScaleText(this.text,
+      {this.style,
+      this.strutStyle,
+      this.maxLines,
+      this.overflow,
+      this.textAlign,
+      this.textDirection,
+      this.maxScaleFactor});

```

Comment:
We should definitely make the `maxScaleFactor` defaults to `1.3` since there are lot of places using 1.3.
And the constructor needs a pretty format.

```suggestion
  const ScaleText(
    this.text, {
    this.style,
    this.strutStyle,
    this.maxLines,
    this.overflow,
    this.textAlign,
    this.textDirection,
    this.maxScaleFactor = 1.3,
  });

```